semantic_release/history/__init__.py

```diff
@@ -25,6 +35,12 @@ def get_current_version():
             re.MULTILINE
         ).group(1)
 
+if (config.has_option('semantic_release', 'versioning_by_tag') and
```

Comment:
If you add the option to defaults.cfg it is not necessary to check if it exists. Can you please add it with false as default value and remove all `has_option` calls?